lib/Chat/ChatManager.php

```diff
@@ -176,6 +176,8 @@ public function addSystemMessage(
 			$comment->setVerb(self::VERB_SYSTEM);
 		}
 
+		$this->setMessageExpiration($chat, $comment);
```

Comment:
This now expires all system messages, including "X added Y", not sure that is expected.
But for expiring call logs it definetly is expected, so I guess it's fine.